### ISSUE TITLE
[BUG] ARW images aren't generating thumbnails nor working

### ISSUE THREAD

**Issue by dlang123** [immich-app/immich#5625]
### The bug

Just installed a brand new 1.90.2 Immich docker, first time using it.  Tried uploading various image types to do a trial run, and some Sony ARW files throw the following errors:

```
[12/10/23 20:29:03] INFO     Booting worker with pid: 2036                      
[12/10/23 20:29:30] INFO     Created in-memory cache with unloading after 300s  
                             of inactivity.                                     
[12/10/23 20:29:30] INFO     Initialized request thread pool with 24 threads.   
[Nest] 1207  - 12/10/2023, 8:29:45 PM   ERROR [JobService] Unable to run job handler (thumbnailGeneration/generate-jpeg-thumbnail): Error: Input file has corrupt header: magickload: Magick: TIFF directory is missing required "ImageLength" field. `MissingRequired' @ error/tiff.c/TIFFErrors/599 (null)
[Nest] 1207  - 12/10/2023, 8:29:45 PM   ERROR [JobService] Error: Input file has corrupt header: magickload: Magick: TIFF directory is missing required "ImageLength" field. `MissingRequired' @ error/tiff.c/TIFFErrors/599 (null)
[Nest] 1207  - 12/10/2023, 8:29:45 PM   ERROR [JobService] Object:
{
  "id": "debecae9-4929-463d-a934-a26fd8f3e459",
  "source": "upload"
}

[Nest] 190  - 12/10/2023, 8:29:47 PM   ERROR [ErrorInterceptor] Failed to serve file
[Nest] 190  - 12/10/2023, 8:29:47 PM   ERROR [ErrorInterceptor] Error: Request aborted
[Nest] 190  - 12/10/2023, 8:29:47 PM   ERROR [ErrorInterceptor] Failed to serve file
[Nest] 190  - 12/10/2023, 8:29:47 PM   ERROR [ErrorInterceptor] Error: Request aborted
```
Sample Image attached

[DSC04964.zip](https://github.com/immich-app/immich/files/13630441/DSC04964.zip)

### The OS that Immich Server is running on

Ubuntu 23.04

### Version of Immich Server

v1.90.2

### Version of Immich Mobile App

1.0.0

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
ghcr.io/imagegenius/immich:1.90.2-alpine
```

### Your .env content

```Shell
DB_PASSWORD=xxx
TYPESENSE_API_KEY=xxx
REDIS_PASSWORD=xxx
PUID=99
HOSTNAME=214def36ac70
S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0
LANGUAGE=en_US.UTF-8
DB_HOSTNAME=xxx
TYPESENSE_VERSION=0.24.1
HOST_CONTAINERNAME=immich
UMASK=022
PWD=/
DB_DATABASE_NAME=immich
NODE_ENV=production
TZ=America/Chicago
DB_PORT=5432
TRANSFORMERS_CACHE=/config/machine-learning
HOME=/root
LANG=en_US.UTF-8
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=00:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.avif=01;35:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:*~=00;90:*#=00;90:*.bak=00;90:*.old=00;90:*.orig=00;90:*.part=00;90:*.rej=00;90:*.swp=00;90:*.tmp=00;90:*.dpkg-dist=00;90:*.dpkg-old=00;90:*.ucf-dist=00;90:*.ucf-new=00;90:*.ucf-old=00;90:*.rpmnew=00;90:*.rpmorig=00;90:*.rpmsave=00;90:
PGID=100
VIRTUAL_ENV=/lsiopy
IMMICH_MEDIA_LOCATION=/photos
S6_VERBOSITY=1
S6_STAGE2_HOOK=/docker-mods
DB_USERNAME=xxx
DISABLE_MACHINE_LEARNING=false
MACHINE_LEARNING_WORKERS=1
REDIS_HOSTNAME=xxx
TERM=xterm
HOST_OS=Unraid
IMMICH_MACHINE_LEARNING_URL=http://127.0.0.1:3003
SHLVL=1
MACHINE_LEARNING_WORKER_TIMEOUT=120
REDIS_PORT=6379
TYPESENSE_HOST=127.0.0.1
MACHINE_LEARNING_CACHE_FOLDER=/config/machine-learning
SERVER_PORT=8080
PATH=/lsiopy/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOST_HOSTNAME=xxx
TYPESENSE_DATA_DIR=/config/typesense
DISABLE_TYPESENSE=false
_=/usr/bin/env
```

### Reproduction steps

```bash
1.Click upload on gui
2.Select ARW file to upload
3.Get bug
...
```

**Comment by alextran1502** [comment]
a regression, confirmed

**Comment by bo0tzz** [comment]
Duplicate of #5605, already solved in #5509

**Comment by mertalev** [comment]
The regression actually seems to be caused by the libvips bump making its way into the AIO image early, so this is an open issue for the time-being.

**Comment by martabal** [comment]
This flavor of AIO has basic support for RAW files, I recommend you to use the `:noml` version instead.

**Comment by martabal** [comment]
The AIO has 3 flavors : 
- the normal version (with the `:latest` tag)
- the noml version (stands for no machine learning, with the `:noml` tag)
- the alpine version (with the `:alpine` tag)

The normal and noml versions use the same scripts to build libraw, imagemagick and libvips as the official Immich containers; they are the same to avoid issues like this. The alpine version instead use the edge repo for these dependencies. It makes it a very lightweight Immich version but has issues with hardware transcoding and raw files.

**Comment by dlang123** [comment]
> The AIO has 3 flavors :
> 
> * the normal version (with the `:latest` tag)
> * the noml version (stands for "no machine learning", with the `:noml` tag)
> * the alpine version (with the `:alpine` tag)
> 
> The normal and noml versions use the same scripts to build libraw, imagemagick and libvips as the official Immich containers, to avoid issues like this. The alpine version instead use the edge repo for these dependencies. It makes it a very lightweight Immich version but has issues with hardware transcoding and raw files.

Actually I am using the latest I believe even though I had put up alpine originally but is actually baseimage-ubuntu:lunar

**Comment by mertalev** [comment]
This is fixed now with #5705.
